rpc_new/client.ts

```diff
@@ -26,13 +46,28 @@ export class Client<DiscoveryValue, SendE, InternalE, CloseE> {
         const pendingCall = this.pendingCalls[id]!;
         pendingCall.resolve(e);
         delete this.pendingCalls[id];
+        this.pendingSubscriptions[id]?.(e);
+        delete this.pendingSubscriptions[id];
+      }
+      for (const id in this.activeSubscriptions) {
+        this.activeSubscriptions[id]?.(e);
+        delete this.activeSubscriptions[id];
```

Comment:
as long as the provider communicate errors through the listeners, this will cover https://github.com/paritytech/capi/issues/245